css/styles.css

```diff
@@ -41,11 +41,13 @@
 /* @group Columns */
 
 .ab-wrap {
-    margin: 20px 0 0 -30px;
+    margin: 0 0 0 -10px;
+    padding: 20px 0 0 0;
+    text-rendering: optimizeLegibility;
 }
 .ab-column {
     float: left;
-    margin: 0 0 0 30px;
+    margin: 0 0 0 10px;
```

Comment:
This, again, is not changing the amount of CSS.
`margin: 0 0 0 10px;` restores the original state in which the columns are aligned with the headline, as the following screenshots illustrate.
![column-margin-01](https://cloud.githubusercontent.com/assets/1038789/22629725/347f0234-ebec-11e6-848f-58b9d990ad25.png)
 **EDIT** With changing `.ab-column`s margin-left to -30px this actually makes some more sense.